index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Aretha Franklin Bio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Aretha Franklin</h1>
+      <p>The woman who sangs Respect</p>
+      <figure id="img-div">
+        <img src="./img/Aretha.png" width="" alt="image of Aretha Franklin" id="image">
+        <figcaption id="img-caption">Aretha Franklin best singer of Blues in the World!</figcaption>
+      </figure>
+      <section id="tribute-info">
+        <p>Aretha Louise Franklin was born on March 25, 1942 (August 16, 2018) was an American singer, songwriter and pianist.[2] Referred to as the "Queen of Soul", she has twice been placed ninth in Rolling Stone's "100 Greatest Artists of All Time". With global sales of over 75 million records, Franklin is one of the world's best-selling music artists.
```

Comment:
Please kindly give style to this paragraph, it is important to use text-align: justify, line-height, and word-spacing to enhance and deliver a beautiful and clean code.